src/super_gradients/common/plugins/deci_client.py

```diff
@@ -54,9 +61,10 @@ def _get_file(self, model_name: str, file_name: str) -> Optional[str]:
         :return:            Path were the downloaded file was saved to. None if not found.
         """
         try:
-            response = self.lab_client.get_autonac_model_file_link(
+            response, status, response_headers = self.lab_client.get_autonac_model_file_link_with_http_info(
```

Comment:
I assume we need to update our pro requirements then ?

Reply:
According to the versions in pro requirements (deci-lab-client==4.17.0) - it's already there. 
In fact get_autonac_model_file_link using get_autonac_model_file_link_with_http_info under the hood. So no version update needed IMO

Reply:
Updated deci-lab-client to 4.29 since there was some import error for BodyRegisterUserArchitecture request model.